web/src/settings_components.js

```diff
@@ -681,6 +681,23 @@ function enable_or_disable_save_button($subsection_elem) {
         disable_save_btn = should_disable_save_button_for_time_limit_settings(time_limit_settings);
     } else if ($subsection_elem.attr("id") === "org-other-settings") {
         disable_save_btn = should_disable_save_button_for_jitsi_server_url_setting();
+        const $button_wrapper = $subsection_elem.find(".subsection-changes-save");
+        const is_tippy = $button_wrapper.get(0)._tippy;
+        if (disable_save_btn) {
+            if (!is_tippy) {
+                initialize_disable_btn_hint_popover(
+                    $button_wrapper,
+                    $t({defaultMessage: "Cannot save invalid Jitsi server URL."}),
+                );
+                $button_wrapper.get(0)._tippy.setProps({
+                    placement: "top",
```

Comment:
To do this in a better way, we can pass `opts` object to `initialize_disable_btn_hint_popover` and then update the code there to be something like -
```js
const tippy_opts = {
    animation: false,
    hideOnClick: false,
    placement: "bottom",
    ...opts,
};
```